File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int i=0;i<nums.length;i++){
            int op = 0;
            int num = nums[i];
            while(num>0){
                op++;
                // int digit = num%10;
                num/=10;
                
            }
            if(op%2 == 0){
                count++;
            }
        }
        
        return count;
    }
}

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
    let count = 0;
    for(let i=0;i<nums.length;i++){
        let op = 0;
        let num = nums[i];
        while(num!=0){
            op++;
            num = Math.floor(num/10);
        }
        if(op%2 == 0){
            count++;
        }
    }
    return count;
};